test/java/org/opendatakit/briefcase/model/RetrieveAvailableFormsFailedEventTest.java

```diff
@@ -0,0 +1,21 @@
+package org.opendatakit.briefcase.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class RetrieveAvailableFormsFailedEventTest {
```

Comment:
Testing getters here, which is nor very useful. I'd suggest removing this. test class